### PR TITLE
Jenkins setup

### PR DESCRIPTION
**Pull request by CyrilBaah**

### CODE REVIEW

File: jenkins/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+JENKINS_AGENT_SSH_PUBLIC_KEY=
```
Comment (ninadingole) [inline jenkins/.env]: EOL missing

**Comment by ninadingole** [comment]
@CyrilBaah, thanks for the PR. Could you please remove the `.DS_Store` files from the PR, everything looks good; I am happy to merge

File: jenkins/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      - /var/run/docker.sock:/var/run/docker.sock
+  agent:
+    image: jenkins/ssh-agent:jdk11
+    container_name: jenkins_agent1
+    privileged: true
+    user: root
+    expose:
+      - 22
+    environment:
+      - JENKINS_AGENT_SSH_PUBKEY=${JENKINS_AGENT_SSH_PUBLIC_KEY}
```
Comment (ninadingole) [inline jenkins/docker-compose.yml]: EOL missing

File: jenkins/.env (hunk 1 above)
Comment (CyrilBaah) [reply jenkins/.env]: Ok, would make the changes

**Review by CyrilBaah — COMMENTED** [review]
Thank you for the feedback, suggestion has been worked on

**Review by ninadingole — APPROVED** [review]

**Comment by ninadingole** [comment]
Thank you so much for your contribution 🎉

**Pull request merged**